corehq/apps/registration/static/registration/js/user_login_form.js

```diff
@@ -0,0 +1,147 @@
+hqDefine('registration/js/user_login_form', [
+    'jquery',
+    'knockout',
+    'hqwebapp/js/initial_page_data',
+    'hqwebapp/js/assert_properties',
+    'hqwebapp/js/utils/email',
+    'hqwebapp/js/knockout_bindings.ko',
+], function (
```

Comment:
is the long term plan that this replaces the current login form? or we'll maintain both indefinitely?

Reply:
I think we need to have both because not all environments want to enforce SSO by default. If we decide that this is a non-issue, it's easy enough to remove the flag and we're on our way.

Reply:
it doesn't replace but adds on to `registration/js/login`. I will likely need to use it for the login form when a session has timed out as well...so this isn't the only place it will be used.